static/src/js/components/SpatialDisplay/__tests__/SpatialDisplay.test.js

```diff
@@ -205,406 +196,537 @@ describe('SpatialDisplay component', () => {
+      const inputVal = '0,95'
+
+      setup({
+        manuallyEntering: 'marker',
+        pointSearch: [inputVal]
+      })
+
+      const input = screen.queryByTestId('spatial-display_point')
 
-      const input = '95,0'
+      expect(input.value).toEqual(inputVal.split(',').reverse().join(','))
 
-      const result = enzymeWrapper.instance().validateCoordinate(input)
+      const errors = screen.queryAllByTestId('filter-stack-item__error')
+      expect(errors).toHaveLength(1)
 
-      expect(result).toEqual('Latitude (95) must be between -90 and 90.')
+      expect(errors[0].innerHTML).toEqual('Latitude (95) must be between -90 and 90.')
     })
 
     test('returns an error for a coordinate an invalid longitude', () => {
-      const { enzymeWrapper } = setup()
+      const inputVal = '190,0'
+
+      setup({
+        manuallyEntering: 'marker',
+        pointSearch: [inputVal]
+      })
+
+      const input = screen.queryByTestId('spatial-display_point')
 
-      const input = '0,190'
+      expect(input.value).toEqual(inputVal.split(',').reverse().join(','))
 
-      const result = enzymeWrapper.instance().validateCoordinate(input)
+      const errors = screen.queryAllByTestId('filter-stack-item__error')
+      expect(errors).toHaveLength(1)
 
-      expect(result).toEqual('Longitude (190) must be between -180 and 180.')
+      expect(errors[0].innerHTML).toEqual('Longitude (190) must be between -180 and 180.')
     })
   })
 
   describe('#validateBoundingBoxCoordinates', () => {
-    test('returns an error coordinates match each other.', () => {
-      const { enzymeWrapper } = setup()
-      const input = '-77.119,38.791'
+    test('returns an error coordinates match each other.', async () => {
+      userEvent.setup()
+
+      const inputVal = '38.791,-77.119'
+
+      setup({
+        manuallyEntering: 'rectangle'
+      })
+
+      const swInput = screen.queryByTestId('spatial-display_southwest-point')
+      await userEvent.click(swInput)
+      await userEvent.type(swInput, inputVal)
+
+      const neInput = screen.queryByTestId('spatial-display_northeast-point')
+      await userEvent.click(neInput)
+      await userEvent.type(neInput, inputVal)
+
+      const updatedSwInput = screen.queryByTestId('spatial-display_southwest-point')
+      const updatedNeInput = screen.queryByTestId('spatial-display_northeast-point')
 
-      const boundingBoxResult = enzymeWrapper.instance()
-        .validateBoundingBoxCoordinates([input, input])
+      expect(updatedSwInput.value).toEqual(inputVal)
+      expect(updatedNeInput.value).toEqual(inputVal)
 
-      expect(boundingBoxResult).toEqual('SW and NE points contain matching coordinates. Please use point selection instead.')
+      const errors = screen.queryAllByTestId('filter-stack-item__error')
+      expect(errors).toHaveLength(1)
+
+      expect(errors[0].innerHTML).toEqual('SW and NE points contain matching coordinates. Please use point selection instead.')
```

Comment:
```suggestion
      expect(screen.getByText('SW and NE points contain matching coordinates. Please use point selection instead.')).toBeInTheDocument()
```
I think this is the same but, we can get rid of data-test-ids for better accessibility on our elements